src/LockupNFTDescriptor.sol

```diff
@@ -67,10 +67,21 @@ contract LockupNFTDescriptor is ILockupNFTDescriptor {
         vars.lockup = ISablierLockup(address(lockup));
         vars.lockupModel = mapSymbol(lockup);
         vars.lockupStringified = address(lockup).toHexString();
-        vars.asset = address(vars.lockup.getAsset(streamId));
-        vars.assetSymbol = safeAssetSymbol(vars.asset);
+        vars.tokenSymbol = safeTokenSymbol(vars.token);
         vars.depositedAmount = vars.lockup.getDepositedAmount(streamId);
 
+        // Load the token's address based on the contract version.
+        if (vars.lockupModel.equal("Sablier Lockup")) {
+            // For the latest version of the Lockup contract, use the `getToken` function.
+            vars.token = address(vars.lockup.getToken(streamId));
+        }
+        // For older versions of the Lockup contract, use the `getAsset` function.
```

Comment:
Instead of saying "older versions" lets be explicit and mention that versions prior to v2.0.0